lib/import-notebook.js

```diff
@@ -47,25 +69,39 @@ export function importNotebook(event?: atom$CustomEvent) {
         return;
       }
 
-      _loadNotebook(filename);
+      _loadNotebook(
+        filename,
+        atom.config.get("Hydrogen.importNotebookResults")
+      );
     }
   );
 }
 
-export async function _loadNotebook(filename: string) {
+/**
+ * Reads the given notebook file and coverts it to a text editor format with
+ * Hydrogen cell breakpoints. Optionally after opening the notebook, it will
+ * also load the previous results and display them.
+ *
+ * @param {String} filename - Path of the file.
+ * @param {Boolean} importResults - Decides whether to display previous results
+ */
+export async function _loadNotebook(
+  filename: string,
+  importResults: boolean = false
+) {
   let data;
   let nb;
   try {
     data = await readFileP(filename);
-    nb = parseNotebook(data);
+    nb = fromJS(JSON.parse(data));
   } catch (err) {
     if (err.name === "SyntaxError") {
       atom.notifications.addError("Error not a valid notebook", {
         detail: err.stack
       });
     } else {
       atom.notifications.addError("Error reading file", {
-        detail: err.message
+        detail: err
```

Comment:
What's the content of `err` ?
If it's markdown string, we may want to use `description` instead of `detail`.

Reply:
I couldn't know since its a generic catch all.

Reply:
So, you found some cases that `err.message` is undefined or something like that ?

Reply:
I am pretty sure it was there before me I just removed the message portion because err should have a built-in to string that represents what the user should see,